### ISSUE TITLE
Add support for google_compute_target_ssl_proxy

### ISSUE THREAD

**Issue by vannicktrinquier** [GoogleCloudPlatform/terraform-validator#1214]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave _+1_ or _me too_ comments; they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If the issue is assigned to a user, that user is claiming responsibility for the issue.

### Description
Terraform vet does not have yet support for the resource targetHttpsProxy and targetSslProxy. Support of those resources can be very beneficial to create policy allowing creation of HTTPS/SSL LB (Target Proxy) using only restricted SSL policies

```
resource type cannot be converted for CAI-based policies: google_compute_target_ssl_proxy. For details, see https://cloud.google.com/docs/terraform/policy-validation/create-cai-constraints#supported_resources].
```

### New or Affected Resource(s)
* google_compute_target_ssl_proxy

### Resource converter exists?

- [google_compute_target_ssl_proxy has a resource converter](https://github.com/GoogleCloudPlatform/terraform-validator/blob/main/converters/google/resources/compute_target_ssl_proxy.go)

**Comment by vannicktrinquier** [comment]
Planning to work on it

**Comment by vannicktrinquier** [comment]
Resolved with #1245
